src/icons/logo-grey.svg

```diff
@@ -1,22 +1,9 @@
-  <path fill="#adadad" d="M114.4 172.9v-.5c-1.9.1-3.6.2-5.2.2h-.5c-4.2 0-8.3-1.8-11.2-4.8-2.9-3-4.5-7.2-4.4-11.4.3-8.5 1.6-16.2 3.7-23-25.4-8.8-48.4-22.7-67.8-40.5-11.8 20.8-18.5 44.9-18.5 70.5 0 14.9 2.3 29.4 6.5 42.9 25.8-12.7 54.8-19.9 85.5-20.1l11.9-13.3z"/>
-  <path fill="#adadad" d="M113.7 138.5c-1.7-.4-3.4-.9-5-1.3-1.7 5.8-2.9 12.4-3.1 19.8-.1 1.8 1.4 3.4 3.3 3.4 1.6 0 3.7 0 6-.2v-.7c0-7.2-.4-14.2-1.2-21z"/>
-  <path d="M108.6 110.3c-6.4-24.7-17.5-47.6-32.3-67.5C56.8 55.4 40.5 72.7 29.1 93c19.4 17.8 42.4 31.7 67.8 40.5 2.7-9.1 6.8-16.7 11.7-23.2zm3.7 17.6c-1.4 2.8-2.6 5.9-3.6 9.2 1.7.5 3.3.9 5 1.3-.4-3.5-.8-7-1.4-10.5z" fill="#a1a1a1"/>
-  <path fill="#989898" d="M283.4 101.3c-.6-1.2-1.2-2.3-1.8-3.5-.9 2.3-1.9 4.7-3 7.1l4.8-3.6zm-69-48.4c-.2.1-.4.1-.6.2-1 7.4-1.5 14.8-1.7 22.4 8.3-4.3 18.8-3 25.8 4 8.6 8.6 8.6 22.7 0 31.3-6 6-14.7 7.8-22.3 5.4 1.2 6.4 2.8 12.7 4.6 18.8 12-3.8 23.4-8.7 34.2-14.6 10.4-13.1 15.8-25.9 18.5-37.2-10.3-15.3-23.5-28.4-38.7-38.7-6.3 1.5-13.2 3.9-20.3 7.5 0 .2 0 .4-.1.6.3.1.5.2.6.3z"/>
-  <path fill="#989898" d="M213.9 52.6c0 .2 0 .4-.1.5.2-.1.4-.1.6-.2-.1-.1-.3-.2-.5-.3z"/>
-  <path fill="#b1b1b1" d="M283.4 101.3c-1.6 1.2-3.1 2.4-4.7 3.6-5.4 11.9-13.9 24.5-26.9 36.8L237 156.4c-.6 3.9-1.5 8.3-2.9 12.8 11.9 22.9 28.2 43.1 47.7 59.5 10-19.6 15.7-41.8 15.7-65.3 0-22.2-5.1-43.3-14.1-62.1zm-57.8 49.1l17.5-17.5c4.3-4.1 8.1-8.3 11.4-12.4-10.8 5.9-22.3 10.8-34.2 14.6 1.5 5.2 3.3 10.3 5.3 15.3z"/>
-  <g opacity=".08">
-    <path fill="#fff" d="M140.9 189.9l9.9-8.7c-3.4-.1-7.1-1.5-10-4.4-2.9-2.9-4.4-6.6-4.4-10l-17.7 20.1-7 7.9-60.5 68.6-21.4 24.3L54 266.4l50.6-44.6 36.3-31.9z"/>
-    <path fill="#fff" d="M184.6 74.4l-17.9 17.9s-38.5 2.7-54.4 35.6c-1.4 2.8-2.6 5.9-3.6 9.2-1.7 5.8-2.9 12.4-3.1 19.8-.1 1.8 1.4 3.4 3.3 3.4 1.6 0 3.7 0 6-.2 7.6-.6 18.2-2.8 25-9.7 0 0-3.3 8.1 3.2 12.4 1.9 1.2 3.8 2.5 5.4 4.1l2.1 2.1c1.6 1.6 2.9 3.5 4.1 5.4 4.2 6.5 12.4 3.2 12.4 3.2-4.6 4.6-7 10.7-8.4 16.6-1.3 5.7-1.5 11.1-1.5 14.4 0 1.8 1.5 3.3 3.4 3.3 11.1-.4 20.3-2.7 28-6.2 33.9-15.6 36.7-54.9 36.7-54.9l.4-.4 17.5-17.5c4.3-4.1 8.1-8.3 11.4-12.4 10.4-13.1 15.8-25.9 18.5-37.2 5.2-22.1-.4-38.3-.4-38.3s-16.1-5.6-38.3-.4c-6.3 1.5-13.2 3.9-20.3 7.5-9.7 4.8-19.6 11.9-29.5 22.3zm53.4 5.1c8.6 8.6 8.6 22.7 0 31.3-6 6-14.7 7.8-22.3 5.4-3.3-1-6.4-2.8-9-5.4-8.6-8.6-8.6-22.7 0-31.3 1.6-1.6 3.5-3 5.5-4 8.3-4.3 18.8-3 25.8 4z"/>
-    <path fill="#fff" d="M283.4 101.3c-.6-1.2-1.2-2.3-1.8-3.5-.9 2.3-1.9 4.7-3 7.1-5.4 11.9-13.9 24.5-26.9 36.8L237 156.4c-.6 3.9-1.5 8.3-2.9 12.8-4.7 15.2-14.7 32.4-32.5 43.5-10.6 6.6-23.9 11-40.7 11.6h-.5c-8.6 0-15.6-7-15.6-15.6 0-1.8 0-3.8.2-6l-51.4 45.2-30.3 26.7c2.9 2.3 5.8 4.6 8.8 6.7 23.2 16.2 51.4 25.6 81.9 25.6 41 0 78-17.2 104.2-44.8 9.4-9.9 17.4-21.1 23.6-33.4 10-19.6 15.7-41.8 15.7-65.3 0-22.2-5.1-43.3-14.1-62.1z"/>
-  </g>
+<svg xmlns="http://www.w3.org/2000/svg" id="Layer_1" viewBox="0 0 460.4 500">
+  <style>
+    .st0{fill:none;stroke:#fff;stroke-width:4.2591;stroke-miterlimit:10}.st1{fill:#fff}
```

Comment:
This class name already exists in a couple other icons. I don't think they're actually used, but generally we should still be careful about these names.